src/Conversion/ONNXToTorch/NN/AddOp.cpp

```diff
@@ -0,0 +1,134 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------- AddOp.cpp - ONNX Op Transform ------------------===//
+//
+// Copyright 2019-2020 The IBM Research Authors.
+//
+// ========================================================================
+//
+// This file implements a combined pass that dynamically invoke several
+// transformation on ONNX ops.
+//
+//===-----------------------------------------------------------------===//
+
+#include <fstream>
+#include <iostream>
+#include <set>
```

Comment:
NIT: are these used?

Looking at the other include files, I find it surprising that some of them are needed (e.g. MD5.h, KrnlOps.hpp).

Reply:
Resolved